### ISSUE TITLE
Guided Tour

### ISSUE THREAD

**Issue by curran** [vizhub-core/vizhub-issue-tracker#698]
From @stushurik :

Idea: add tips or tour to vizhub. As a user I would like to explore vizhub features and have hints with help. Tour might shows users available features, explain how to use it (Might be shown only once on first sign in). Also it would be nice to have "?" icons that active help tooltips
